### ISSUE TITLE
CI: Changelog reminder does not work (for PRs only?)

### ISSUE THREAD

**Issue by d-Rickyy-b** [d-Rickyy-b/certstream-server-go#45]
Somehow the changelog reminder GH action wasn't able to run. It says `Error: Resource not accessible by integration`.

See: https://github.com/d-Rickyy-b/certstream-server-go/actions/runs/12063604918/job/33670710888?pr=44

**Comment by meinder-a** [comment]
Hi Rickyy,

I think it's okay, looks like the changelog reminder action failed simply because no changes were made to the CHANGELOG.md file. Since it's just a reminder, I believe everything's working as intended here.

https://github.com/d-Rickyy-b/certstream-server-go/pull/44/files

```
    message: @x123 We couldn't find any modification to the CHANGELOG.md file. If your changes are not suitable for the changelog, that's fine. Otherwise please add them to the changelog!

```

**Comment by d-Rickyy-b** [comment]
Hi @meinder-a,

the message you quoted would have been posted to the PR because precisely there were no modifications of the CHANGELOG.md file. The github action isn't able to access the "resource". Maybe that's the changelog file it can't access? I don't know exactly, since I didn't have the time to actually check this. I might have a look later or next week.

**Comment by d-Rickyy-b** [comment]
Should be fixed by aa05b98fbf72497fbd4f0319d62b80e967bdbbcc
